docs/application/dotnet/guides/user-interface/nui/nui-components/DatePicker.md

```diff
@@ -0,0 +1,71 @@
+# DatePicker
+
+DatePicker is a class which provides a function that allows the user to select a date through a scrolling motion by expressing the specified value as a list. DatePicker expresses the current date using the locale information of the system. Year range is 1970~2038 (glibc time_t struct min, max value)
+
+![DatePicker](../../screenshots/DatePicker.png)
+
+## Add namespace
+
+To implement datepicker, include `Tizen.NUI.Components` and namespace in your application:
+
+```xaml
+xmlns:comp="clr-namespace:Tizen.NUI.Components;assembly=Tizen.NUI.Components"
+```
+
+## Create with property
+
+To create a DatePicker, follow these steps:
+
+1. Create DatePicker in XAML:
+
+    ```xaml
+    <comp:DatePicker WidthSpecification="-1" HeightSpecification="-1"/>
+    ```
+
+2. Set the DatePicker format property:
+
+    ```xaml
+    <comp:DatePicker x:Name="picker" Format="yyyy/MM/dd"/>
+    ```
+
+The following output is generated when the DatePicker is created using property:
+
+![DatePicker](../../screenshots/DatePicker.png)
+
+## Respond to value changed event
+
+When you touch or pan a DatePicker, the DatePicker instance receives a value changed event.
+You can declare the value changed event handler as follows:
+
+```xaml
+<comp:DatePicker x:Name="picker" ValueChanged="OnValueChanged"/>
+```
+
+```csharp
+private void OnValueChanged(object sender, ValueChangedEventArgs args)
+{
+    // Do something in response to DatePicker click
+}
+```
+
+## Respond to date changed event
+
+When you change date in DatePicker, the DatePicker instance receives a date changed event.
+You can declare the date changed event handler as follows:
+
+```xaml
+<comp:DatePicker x:Name="picker" DateChanged="OnDateChanged"/>
+```
+
+```csharp
+private void OnDateChanged(object sender, DateChangedEventArgs args)
+{
+    // Do something in response to DatePicker click
+}
+```
+
+## Related information
+
+- Dependencies
+  -   Tizen 6 and Higher 

```

Comment:
Global. At here, Tizen Platform version is written.

Would you please use Tizen Platform version here?
As DatePicker is added since Tizen 6.5, I guess 6.5 should be used at here.

Tizen 6.5 and Higher